pyccel/parser/semantic.py

```diff
@@ -3275,50 +3290,23 @@ def _visit_FunctionDef(self, expr, **settings):
 
             # Collect the modified objects
             lhs_assigns   = [a.lhs for a in assigns]
-            modified_args = [func_arg for f in calls
-                                for func_arg, inout in zip(f.args,f.funcdef.arguments_inout) if inout]
+            modified_args = [call_arg.value for f in calls
+                                for call_arg, func_arg in zip(f.args,f.funcdef.arguments) if func_arg.inout]
```

Comment:
Can you please add a comma between the arguments of `zip`?

Reply:
Do you mean a space?

Reply:
Yes! I meant a space after the comma

Reply:
Fixed b80b62c3f6a4032a8edfc87cbc1416456e96faa5